queue_services/entity-filer/src/entity_filer/filing_processors/transition.py

```diff
@@ -0,0 +1,87 @@
+# Copyright © 2020 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""File processing rules and actions for the transition of a business."""
+import datetime
+from http import HTTPStatus
+from typing import Dict
+
+import sentry_sdk
+from entity_queue_common.service_utils import QueueException
+from legal_api.models import Business, Filing
+from legal_api.services.bootstrap import AccountService
+
+from entity_filer.filing_processors.filing_components import aliases, business_info, shares
+from entity_filer.filing_processors.filing_components.offices import update_offices
+from entity_filer.filing_processors.filing_components.parties import update_parties
+
+
+def create_affiliation(business: Business, filing: Filing):
```

Comment:
Should this and the affiliation in IA be combined?
Seems like most of the 2 would be similar, except for a couple of extra steps.

Reply:
As discussed, the affiliation should happen outside the filer process and this code is no longer needed